src/illuminant/cct.rs

```diff
@@ -335,137 +335,143 @@ fn duv_interpolate(u: f64, v: f64, imp: usize, imn: usize, dp: f64, dh: f64) ->
     dd - d
 }
 
-#[test]
-fn test_ends() {
-    use approx::assert_ulps_eq;
-    // Temperature  at the low end, should pass...
-    let cct0 = CCT::new(1000.0, 0.0).unwrap();
-    let xyz: XYZ = cct0.try_into().unwrap();
-    let cct: CCT = xyz.try_into().unwrap();
-    assert_ulps_eq!(cct, cct0);
-
-    // Temperature  at the low end, should pass...
-    let cct0 = CCT::new(1E6, 0.0).unwrap();
-    let xyz: XYZ = cct0.try_into().unwrap();
-    let cct: CCT = xyz.try_into().unwrap();
-    assert_ulps_eq!(cct, cct0);
-
-    // Temperature  at the low end, should pass...
-    let cct0 = CCT::new(1005.0, 0.0).unwrap();
-    let xyz: XYZ = cct0.try_into().unwrap();
-    let cct: CCT = xyz.try_into().unwrap();
-    assert_ulps_eq!(cct, cct0, epsilon = 1E-5);
-
-    // Temperature  at the low end, should pass...
-    let cct0 = CCT::new(1E6 - 100.0, 0.0).unwrap();
-    let xyz: XYZ = cct0.try_into().unwrap();
-    let cct: CCT = xyz.try_into().unwrap();
-    approx::assert_abs_diff_eq!(cct, cct0, epsilon = 0.2);
-}
+#[cfg(test)]
+mod tests {
```

Comment:
This PR looks bigger than it really is. Because the addition of this module changed indentation on a large set of lines of code. But it does not really change the code. Just putting it in its own module makes sure stuff in here that is only used during tests are not part of real builds.